### DISCUSSION TITLE
Any GPU support ...Intel UHD 770 Graphics  or Nvidia GeForce??

### DISCUSSION THREAD

**Discussion by avegqman** [roflcoopter/viseron#694] [General]
Is there any support for Intel GPU or Nvidia GPU? etc...

**Comment by roflcoopter** [comment]
Yes there is, for both object detection and camera decoding.
If you have an NVidia GPU i suggest using that.

For object detection you can use `darknet`: https://viseron.netlify.app/components-explorer/components/darknet#hardware-acceleration
For camera decoding you can use `ffmpeg`: https://viseron.netlify.app/components-explorer/components/ffmpeg#hardware-acceleration

**Comment by bsyomov** [comment]
Ffmpeg also supports video encoding/decoding hardware acceleration on Intel and AMD GPUs(Including built-in) via VAAPI.
